FrameworkSystem/DB/ProxyDB.py

```diff
@@ -596,13 +645,143 @@ def __getPUSProxy(self, userDN, userGroup, requiredLifetime, requestedVOMSAttr=N
       return S_ERROR('Requested DN does not match the obtained one in the PUSP proxy')
     timeLeft = credDict['secondsLeft']
 
-    result = chain.generateProxyToString(lifeTime=timeLeft,
-                                         diracGroup=userGroup)
+    result = chain.generateProxyToString(timeLeft, diracGroup=userGroup)
     if not result['OK']:
       return result
     proxyString = result['Value']
     return S_OK((proxyString, timeLeft))
 
+  def __getDIRACCAProxy(self, fullName, eMail, DN=None, diracGroup=None, proxyProvider='DIRAC_EOSC_CA'):
+
+    def writeUserConfigFile(fileName, fullName, eMail):
```

Comment:
This method deserves an extended explanation

Reply:
It method was deleted. Instead, it uses ProxyProviderFactory().getProxyProvider(proxyProvider)